### PR TITLE
classify whole class name /w modules instead of just class name in grabbers

### PR DESCRIPTION
**Pull request by xnutsive**
Or else `by_symbol(:twitter)` returns module `Twitter`, which has nothing to do with IdsPlease.

### CODE REVIEW

File: lib/ids_please/grabbers.rb [hunk 1]
```diff
@@ -19,8 +19,7 @@ def self.each
     end
 
     def self.by_symbol(sym)
-      klass_name = "#{sym.to_s[0].upcase}#{sym.to_s[1..-1]}"
-      self.const_get(klass_name)
+      "IdsPlease::Grabbers::#{sym.to_s.capitalize}".constantize
```
Comment (gazay) [inline lib/ids_please/grabbers.rb]: It'll not gonna work without rails

Comment (xnutsive) [inline lib/ids_please/grabbers.rb]: Yup. Gotta rewrite this & test that it works correctly.

File: lib/ids_please/grabbers/twitter.rb [hunk 2]
```diff
@@ -1,28 +1,23 @@
...
-          next if v.nil? || v == ''
-          @data[k] = CGI.unescapeHTML(v)
-        end
-        self
-      rescue => e
-        p e
-        return self
-      end
-
+class IdsPlease::Grabbers::Twitter < IdsPlease::Grabbers::Base
```
Comment (gazay) [inline lib/ids_please/grabbers/twitter.rb]: why you need to do this?

**Comment by gazay** [comment]
Fixed without metaprogramming: https://github.com/gazay/ids_please/commit/a3b4e0da531225b66d2a15a24dd7cf8d8904c2d9

**Pull request closed without merging**